model/src/lib.rs

```diff
@@ -760,9 +760,11 @@ impl FundingFee {
     }
 }
 
-#[derive(Copy, Clone, Debug, PartialEq)]
+#[derive(Copy, Clone, Debug, PartialEq, Serialize, Deserialize)]
 pub enum FeeFlow {
+    #[serde(with = "::bdk::bitcoin::util::amount::serde::as_sat")]
     LongPaysShort(Amount),
+    #[serde(with = "::bdk::bitcoin::util::amount::serde::as_sat")]
```

Comment:
🔧 I guess this is not necessary anymore.

Reply:
Removed in [`2c1fea6` (#1981)](https://github.com/itchysats/itchysats/pull/1981/commits/2c1fea64473efb878d23b18c0b8734c66d1277b2)